packages/http-client-csharp/eng/scripts/Test-Packages.ps1

```diff
@@ -6,6 +6,16 @@ param(
     [string] $Filter = "."
 )
 
+function BuildPackage {
+    # Fail if anything written to stderr during build
```

Comment:
Please don't assume that stderrr doesn't mean error happened, many tools including typespec will use stderr for logging.
Every tool that emit warning usually has a way to treat them as error and fail the status code.

Reply:
Thanks, updated the config which already supports failing on warnings.